src/javascript/app/base/socket_general.js

```diff
@@ -31,14 +31,18 @@ const BinarySocketGeneral = (() => {
     const onMessage = (response) => {
         Header.hideNotification('CONNECTION_ERROR');
         let is_available = false;
+        let has_message = false;
```

Comment:
I don't see why we need to declare it false here, and then assign it on line 39. why not just change line 42 to `} else if (response.website_status.message) {`

Reply:
Yeah, that's an artifact of the previous commits, thanks for catching this :+1: